src/worker/lambda/handlerPuller.go

```diff
@@ -21,6 +21,11 @@ import (
 
 var notFound404 = errors.New("file does not exist")
 
+// for regex inside Pull() use
+var handlerNameRegex *regexp.Regexp
```

Comment:
Instead of the 3 variables, can we just have this?

var handlerNameRegex = regexp.MustCompile(`^[A-Za-z0-9\.\-\_]+$`)